### ISSUE TITLE
[imp] allow products to be published at a certain date and time

### ISSUE THREAD

**Issue by bruecksen** [bruecksen/bakeup#507]
At the moment a product of a production day can be published with a boolean flag. It would be nice to allow setting a day and time when the product should be published.

**Comment by bruecksen** [comment]
@madmooose yes that would be nice. Can we have a simple first implemenation that uses the generic approach and then extend it to the other areas when needed? Do you have a specific implementation in mind?

**Comment by madmooose** [comment]
@bruecksen just the examples above
- publish Produktion day 2 Werks before creation
- publish product in a specific date (that is not in the Papst and not after Produktion day
- set a specific vor all products to in Produktion 12 hours before planed date of Produktion of the product (possible for the future)
- add a widget that displays the calculatet time of event for relative date fields
